src/tasks.ts

```diff
@@ -26,7 +26,7 @@ class DenoRunTaskAssistant implements TaskAssistant {
     const permissionTuples = ([
       ["co.gwil.deno.tasks.run.config.allow.read", "--allow-read"],
       ["co.gwil.deno.tasks.run.config.allow.write", "--allow-write"],
-      ["co.gwil.deno.tasks.run.config.allow.network", "--allow-network"],
+      ["co.gwil.deno.tasks.run.config.allow.net", "--allow-net"],
```

Comment:
There's one thing you're missing here, which is updating the config name in `extensions.json`, [L190](https://github.com/sgwilym/nova-deno/blob/main/deno.novaextension/extension.json#L190). This is basically the string which wires up the checkbox in a project's settings with this task assistant.

Change that and we should be good to go!

Reply:
Cool, after this I have another PR coming to fix up a bunch of other config/task related things.